R/exportToJson.R

```diff
@@ -619,9 +621,18 @@ generateDomainMetaReport <- function(conn, dbms, cdmDatabaseSchema, resultsDatab
                                             vocab_database_schema = vocabDatabaseSchema
   )  
   
-  output$MESSAGES <- querySql(conn,queryDomainMeta)
-  jsonOutput = toJSON(output)
-  write(jsonOutput, file=paste(outputPath, "/domainmeta.json", sep=""))  
+  try(
+    output$MESSAGES <- querySql(conn,queryDomainMeta)
+  )
+  if (output$MESSAGES = NULL)

```

Comment:
Hi, sorry, but i realize now that it's an an external library, and I don't htink we want to introduce a new dependency to Achilles. So is.null() is something in the base libary that you could use in this case.